extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -32,12 +32,13 @@ const TIMEOUT_VALUE = 10000;
 export class AmpAdApiHandler {
 
   /**
-   * @param {!AMP.BaseElement} baseInstance
+   * @param {!./amp-ad-3p-impl.AmpAd3PImpl|../../amp-a4a/0.1/amp-a4a.AmpA4A} baseInstance
    * @param {!Element} element
    * @param {function()=} opt_noContentCallback
    */
   constructor(baseInstance, element, opt_noContentCallback) {
-    /** @private {!AMP.BaseElement} */
+
+    /** @private {!./amp-ad-3p-impl.AmpAd3PImpl|../../amp-a4a/0.1/amp-a4a.AmpA4A}*/
```

Comment:
I see. it did work with only `@private`. Hmmm the type check thing confused me so much 😢